### PR TITLE
Add io.gitlab.sdl_jstest.sdl_jstest

### PR DESCRIPTION
**Pull request by Grumbel**
Flathub manifest for https://gitlab.com/sdl-jstest/sdl-jstest

### CODE REVIEW

File: sdl-jstest.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      the joysticks by displaying the events they send or by
+      displaying their current button, axis, hat or ball state.
+      sdl-jstest is especially useful if you want to test your
+      `SDL_LINUX_JOYSTICK` configuration.
+    </p>
+  </description>
+  <url type="homepage">https://sdl-jstest.gitlab.io</url>
+  <screenshots>
+    <screenshot type="default">https://sdl-jstest.gitlab.io/screenshot-0.2.1-1.png</screenshot>
+  </screenshots>
```
Comment (nedrichards) [inline sdl-jstest.appdata.xml]: Can you add version tags and [OARS](https://hughsie.github.io/oars/), please?

**Comment by nedrichards** [comment]
bot, build io.gitlab.sdl_jstest.sdl_jstest

**Comment by nedrichards** [comment]
As you are upstream, is there a reason you don't include the appdata there?

**Comment by nedrichards** [comment]
The appdata isn't being installed quite correctly: https://flathub.org/builds/#/builders/39/builds/249

https://flathub.org/builds/#/builders/39/builds/249/steps/5/logs/stdio has a bunch of debug output that may help you work out why the appstream data isn't being composed together with the icons etc.

**Comment by Grumbel** [comment]
Moved appdata upstream and added OARS and release tags.

**Comment by nedrichards** [comment]
bot, build io.gitlab.sdl_jstest.sdl_jstest

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.gitlab.sdl_jstest.sdl_jstest

**Pull request closed without merging**